src/common/repositories/ContractEntityRepository.ts

```diff
@@ -81,6 +82,13 @@ export default class ContractEntityRepository extends Subject<ContractEntity> {
               EntityType.RedeemIntentConfirmeds,
               EntityType.RedeemProgresseds,
               EntityType.UnstakeProgresseds,
+              // M1 facilitator entities
+              // Deposit and confirm deposit entities
+              EntityType.DeclaredDepositIntents,
+              EntityType.ConfirmedDepositIntents,
+              // Withdraw and Confirm withdraw entities
+              EntityType.DeclaredWithdrawIntents,
+              EntityType.ConfirmedWithdrawIntents,
```

Comment:
Same comment here also. add missing entities since the name has changed.

Reply:
Done